src/index.js

```diff
@@ -1,18 +1,22 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
```

Comment:
- [ ] When I try to compile your app, I get an error due to this line. It says `Routes` is not imported from `react-router-dom`. This is true because in your `package.json` file you have version 5 of react-router-dom `("react-router-dom": "^5.3.0")`, whereas `Routes` only started becoming available from version 6. Kindly fix this.

  ![compile](https://user-images.githubusercontent.com/36542986/173551808-c76683c5-b65d-4296-bb87-9c8ffdf8afa4.jpg)

  ## SUGGESTION
  Delete this line `"react-router-dom": "^5.3.0"` from your `package.json` file, and run:

  ```
  npm i react-router-dom
  ```